### ISSUE TITLE
CI failing

### ISSUE THREAD

**Issue by petemoore** [taskcluster/taskcluster-hooks#60]
See https://github.com/taskcluster/taskcluster-hooks/commits/a0832c72aab1cea1b61e2c7ae1d656e4356587a1

**Comment by djmitche** [comment]
```
  1) API
       getHookStatus
         returns the last run status for triggerHook:
     Error: should NOT have additional properties
----
method:     triggerHook
errorCode:  InputError
statusCode: 400
time:       2018-01-17T03:54:28.100Z
      at node_modules/taskcluster-client/lib/client.js:318:21
      at <anonymous>
      at process._tickDomainCallback (internal/process/next_tick.js:228:7)
  2) API
       triggerHook
         should launch task with the given payload:
     Error: should NOT have additional properties
----
method:     triggerHook
errorCode:  InputError
statusCode: 400
time:       2018-01-17T03:54:28.128Z
      at node_modules/taskcluster-client/lib/client.js:318:21
      at <anonymous>
      at process._tickDomainCallback (internal/process/next_tick.js:228:7)
```

@alexandrasp can you take a look and see what's wrong?

**Comment by alexandrasp** [comment]
@djmitche, trying to find the problem :} 
But I did get @petemoore editions and all test pass.

**Comment by petemoore** [comment]
Fixed in #61
